vnext/PropertySheets/Warnings.props

```diff
@@ -28,6 +28,8 @@
 
   <ItemDefinitionGroup>
     <ClCompile>
+      <!-- /permissive- by default to enforce standards conformance, unless ENABLEPermissive has been set -->
+      <AdditionalOptions Condition="'$(ENABLEPermissive)' == ''">/permissive- %(AdditionalOptions)</AdditionalOptions>
```

Comment:
Why is this in `Warnings.props`?

Reply:
enabling strict C++ warnings seemed like it should go with warnings?